tests/source/string-lit.rs

```diff
@@ -59,3 +59,12 @@ fn issue_1282() {
 // #1987
 #[link_args = "-s NO_FILESYSTEM=1 -s NO_EXIT_RUNTIME=1 -s EXPORTED_RUNTIME_METHODS=[\"_malloc\"] -s NO_DYNAMIC_EXECUTION=1 -s ELIMINATE_DUPLICATE_FUNCTIONS=1 -s EVAL_CTORS=1"]
 extern "C" {}
+
+// #4471 - strings including `\` shaouldnot wrap at the `\`
```

Comment:
fixing typo, plus adding details to clarify the significance of the `\` in these examples. 

```suggestion
// #4471 - strings including escape characters `\` should not wrap at the escape character `\`
```